app/controllers/codelists-management/edit.js

```diff
@@ -0,0 +1,7 @@
+import Controller from '@ember/controller';
+
+export default class CodelistsManagementEditController extends Controller {
+  reset() {
+    this.model.codelist.rollbackAttributes();
```

Comment:
I don't think this rollback is needed since a changeset is used in the form component?